### ISSUE TITLE
Project show for extra-detailed repr

### ISSUE THREAD

**Issue by vreuter** [pepkit/peppy#295]
For interactive use, we want to limit data displayed by `__repr__` since we're interested in more frequent exploratory than debug usage context. To support more detailed inspection, though, we'd like a `.show` available to display additional information

- [ ] ordered config data (leverage behavioral inheritance from `attmap`.)
- [ ] table of samples (as frame)

**Comment by vreuter** [comment]
@nsheff should this be moved to 0.23 so that 0.22 can be released sooner to support looper?

**Comment by nsheff** [comment]
I'll leave it up to you to decide. I'm not sure how much work this will be for you. if this will take more than an hour or two then yes, otherwise go for it... I just want to get these releases done in the next few days...

**Comment by vreuter** [comment]
Yeah I think we should try to release the new versions today or tomorrow, so this should wait.
